CHANGELOG.md

```diff
@@ -1,4 +1,3 @@
-# Changelog
```

Comment:
hmm was this meant to be removed?

Reply:
No, I think something got messed up when it diffed what's in main against this branch